libs/clients/zebpay/client_test.go

```diff
@@ -0,0 +1,73 @@
+//go:build integration && vpn
+// +build integration,vpn
+
+package zebpay
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/brave-intl/bat-go/libs/cryptography"
+	"github.com/google/uuid"
+	"github.com/shopspring/decimal"
+	"github.com/stretchr/testify/suite"
+)
+
+type ZebpayTestSuite struct {
+	suite.Suite
+	signingKey cryptography.PrivateKey
+	apikey     string
+}
+
+func TestZebpayTestSuite(t *testing.T) {
+	suite.Run(t, new(ZebpayTestSuite))
+}
+
+func (suite *ZebpayTestSuite) SetupTest() {
+	apiKey := os.Getenv("ZEBPAY_API_KEY")
+	signingKey := os.Getenv("ZEBPAY_SIGNING_KEY")
```

Comment:
Can we make these separate test env variables so we don't accidentally log the real keys while running tests in the CI if it fails?